### DISCUSSION TITLE
App Check only available for ALL Firebase Components?

### DISCUSSION THREAD

**Discussion by GhaniyKie** [vuejs/vuefire#1476] [Questions and Help] [answered]
Does App Check for Vuefire Nuxt (that configured in `nuxt.config.ts`) applied to all Firebase Components: Like Auth, etc?
If yes, is any configuration to prevent this behavior? Since my Firebase Plan is `Spark Plans` that not including Authentication being checked with App Check, i got 403 Error from this behavior 😄.

Error Logs:
```
FirebaseError: AppCheck: Fetch server returned an HTTP error status. HTTP status: 403. (appCheck/fetch-status-error).
```

**Comment by posva** [comment, marked as answer]
You should be fine using app check and auth. You can see the examples at https://vuefire.vuejs.org/nuxt/deployment.html

**Reply by GhaniyKie** [reply]
Wah, i see...

Thanks for your information 👍
